ares/sfc/ppu-performance/ppu.hpp

```diff
@@ -1,6 +1,7 @@
-struct PPU : Thread, PPUcounter {
+#define PPU PPUPerformance
+
+struct PPU : PPUBase::Implementation, PPUcounter {
   Node::Object node;
-  Node::Video::Screen screen;
```

Comment:
Is it necessary to move screen to the impl? Can't the impl access the screen member on the parent class; this prevents all the cases of having to replace screen with screen(), and gives consistency with other cores.